docker/run.sh

```diff
@@ -10,12 +10,25 @@ cd "$( dirname "${BASH_SOURCE[0]}" )"
 # Setup env variables with VMm's ip
 source ../aws-infrastructure/set-env.sh
 
-# Pull images and tag them on client VM
+# Copy executable script
 scp -i ../aws-infrastructure/aws_ssh_key -oStrictHostKeyChecking=no ./upload.py ubuntu@$PUBLIC_CLIENT_IP_ADDR:/home/ubuntu/upload.py
-ssh -i ../aws-infrastructure/aws_ssh_key -oStrictHostKeyChecking=no ubuntu@$PUBLIC_CLIENT_IP_ADDR <<'ENDSSH'
-set -x
-/home/ubuntu/upload.py
-ENDSSH
+scp -i ../aws-infrastructure/aws_ssh_key -oStrictHostKeyChecking=no ./upload.py ubuntu@$PUBLIC_SERVER_IP_ADDR:/home/ubuntu/upload.py
```

Comment:
@Sammers21 can we just start ssh agent and add this key to agent instead of repeating key argument?
```bash
eval $(ssh-agent)
ssh-add ../aws-infrastructure/aws_ssh_key
scp -oStrictHostKeyChecking=no ./upload.py ubuntu@$PUBLIC_SERVER_IP_ADDR:/home/ubuntu/upload.py
```